misc/scripts/bwrap.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+
+#     ____                  __        ____
+#    / __ \____ ___________/ /_____ _/ / /
+#   / /_/ / __ `/ ___/ ___/ __/ __ `/ / /
+#  / ____/ /_/ / /__(__  ) /_/ /_/ / / /
+# /_/    \__,_/\___/____/\__/\__,_/_/_/
+#
+# Copyright (C) 2020-present
+#
+# This file is part of Pacstall
+#
+# Pacstall is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, version 3 of the License
+#
+# Pacstall is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Pacstall. If not, see <https://www.gnu.org/licenses/>.
+
+function safe_source() {
+    local input="${1}"
+    mkdir /tmp/pacstall 2>/dev/null
+    safeenv="$(sudo mktemp -p "${SRCDIR}")"
+    sudo chmod +r "$safeenv"
+    bwrapenv="$(sudo mktemp -t "${SRCDIR}/bwrapenv.XXXXXXXXXX")"
+    sudo chmod +r "$bwrapenv"
+    export bwrapenv
+
+    tmpfile="$(sudo mktemp -p "${SRCDIR}")"
+    echo "#!/bin/bash -ae" | sudo tee "$tmpfile" > /dev/null
```

Comment:
I don't like the `-e` flag. Causes *way* too many problems that all pacscripts are checked for either way.